views/partials/banner_breach.hbs

```diff
@@ -13,6 +13,6 @@
     <span class="breach-info"><span class="medium">Compromised Data</span> {{breachDataClasses breach.meta.DataClasses }}</span>
     <span class="breach-info"><span class="medium">Compromised Accounts</span> {{localeString breach.meta.PwnCount }}</span>
     <span class="breach-info">{{{ breach.meta.Description }}}</span>
-    <span class="source-info">Breach data provided by<span class="demi"> Have I Been Pwned?</span></span>
+    <span class="source-info">Breach data provided by<a class="demi" href="https://www.haveibeenpwned.com" alt="Have I Been Pwned"> Have I Been Pwned</a></span>
```

Comment:
Not technically a problem, per se, and barely even a nit, but...

It feels odd that we have a leading space at the beginning of the `<a>` tag's contents. Since we have a `text-decoration: none` in the CSS, it's a bit moot, but if we eventually refactor that to add default underlines or whatever, the link will have a leading space. Same with the link below. It almost may be nice to refactor this out into some "attibution" partial [eventually], as its fairly duplicated code (apart from the `<a class="demi">` versus `<a class="demi compromised">`).